### DISCUSSION TITLE
Comments

### DISCUSSION THREAD

**Discussion by HemboPagi** [directus/directus#12499] [Archive]
Hello, my question is about comments permissions . It seems that by default (uless you are admin) one sees only his/hers comments. I mentioned user in comment. User got notified about it but can not actually see it. When I gave all access to my test role to Directus Activity collection, then all the comments were displayed.

How can i set it up in a way that user can see his/her own comments and comments with mentions as well?

Thank you in advance
Hembo

**Comment by benhaynes** [comment]
Hmm, that's surprising — I would assume the "Recommended App Permissions" would allow for reading/viewing comments. @rijkvanzanten, are we being more restrictive here by default?

**Reply by rijkvanzanten** [reply]
It's the _and comments with mentions as well?_ part that makes it a little more tricky. The permissions are based on read access to directus_activity, which in turn is based on the read access for the particular collection/item you're trying to access
